modules/core/23-commitment/types/merkle_test.go

```diff
@@ -27,9 +27,6 @@ func (suite *MerkleTestSuite) TestVerifyMembership() {
 	proof, err := types.ConvertProofs(res.ProofOps)
 	require.NoError(suite.T(), err)
 
-	suite.Require().NoError(proof.ValidateBasic())
```

Comment:
Why were these removed?

Reply:
This is setup code. `proof.ValidateBasic()` was equivalent to `proof.Empty()`. These tests are testing `TestVerifyMembership` and `TestVerifyNonMembership` not `ConvertProofs`. I could add back a check that the returned proof is not empty, but it's just going to see if the setup code worked correctly